src/agent/scm/lib/tfvcwrapper.ts

```diff
@@ -134,27 +151,11 @@ export class TfvcWrapper extends events.EventEmitter {
 
         return msg;
     }
-
-    private _exec(cmd: string, args: string[], options?: ITfvcExecOptions): Q.Promise<number> {
-        if (this.tfPath === null) {
-            return this._getTfNotInstalled();
-        }
-
+    
+    private _getToolRunner(cmd: string, args: string[]) {
         var tf = new tl.ToolRunner(this.tfPath);
```

Comment:
You didn't change how this.tfPath is set.  I sent you the other commit on what I did with node.  It needs to look in {agent}/runtime/tee and prefer that one and fall back to which('tf').  We do not set internal runtime tools to the path since those are our internal tools.  For example, an node task would use the one they install but our agent has its version it prefers to run with.